src/services/users.service.js

```diff
@@ -0,0 +1,48 @@
+const { createId } = require('../utils/id.servise');
+
+let users = [];
+
+const resetUsers = () => {
+  users = [];
+};
+
+const getUsers = () => {
+  return users;
+};
+
+const getUser = (id) => {
+  return users.find((elem) => elem.id === id);
+};
+const addUser = (elem) => {
+  const elemWithId = {
+    id: createId(users),
+    ...elem,
+  };
+
+  users.push(elemWithId);
+
+  return elemWithId;
+};
+const updateUser = (newElem) => {
```

Comment:
The `updateUser` function does not return any value. Consider returning the updated user or a status to indicate the update was successful, depending on how this function is used elsewhere.